contracts/src/contracts/utils.py

```diff
@@ -555,3 +567,28 @@ def closeAuction(client: AlgodClient, appID: int, closer: Account):
     client.send_transaction(signedDeleteTxn)
 
     waitForTransaction(client, signedDeleteTxn.get_txid())
+
+
+def fundAccount(client: AlgodClient, funder: Account, recipient: Account, amount: int):
```

Comment:
Not currently used. Added this to support a scenario more closely matching how we will do things in AlgoMart's Node.js code (funding wallet sends funds to generated creator account, then creator account creates the smart contract). Leaving it in here in case we decide to use it.